diff -ur zeropad.bak/Linux/linux.cpp zeropad/Linux/linux.cpp
--- zeropad.bak/Linux/linux.cpp	2008-08-10 13:33:07.000000000 -0400
+++ zeropad/Linux/linux.cpp	2008-08-10 13:33:34.000000000 -0400
@@ -20,7 +20,6 @@
 #include <gtk/gtk.h>
 #include <pthread.h>
 
-#define JOYSTICK_SUPPORT
 #ifdef JOYSTICK_SUPPORT
 #include <SDL/SDL.h>
 #endif
diff -ur zeropad.bak/configure.ac zeropad/configure.ac
--- zeropad.bak/configure.ac	2008-08-10 13:33:07.000000000 -0400
+++ zeropad/configure.ac	2008-08-10 14:09:07.000000000 -0400
@@ -73,16 +73,23 @@
 
 AC_CHECK_FUNCS([ _aligned_malloc _aligned_free ], AC_DEFINE(HAVE_ALIGNED_MALLOC))
 
-AC_CHECK_LIB(SDL,SDL_Init,[])
+AC_ARG_ENABLE(joystick, AC_HELP_STRING([--disable-joystick], [Disable joystick support]),
+ joystick=$enableval,joystick=yes)
+if test "x$joystick" == xyes
+then
+ AC_CHECK_LIB(SDL,SDL_Init,[])
+ AC_CHECK_HEADER([SDL/SDL.h], [AC_DEFINE(JOYSTICK_SUPPORT,1)])
+ LIBS+="$(pkg-config --libs sdl) "
+ CFLAGS+="$(pkg-config --cflags sdl) "
+ CXXFLAGS+="$(pkg-config --cflags sdl) "
+fi
 
 dnl gtk
 AC_MSG_CHECKING(gtk2+)
 AC_CHECK_PROG(GTK_CONFIG, pkg-config, pkg-config)
-LIBS+=$(pkg-config --libs gtk+-2.0 sdl)
-CFLAGS+="$(pkg-config --cflags gtk+-2.0 sdl) "
-CXXFLAGS+="$(pkg-config --cflags gtk+-2.0 sdl) "
-
-dnl AC_CHECK_HEADER([SDL/SDL.h], [AC_DEFINE(JOYSTICK_SUPPORT,1)])
+LIBS+="$(pkg-config --libs gtk+-2.0) "
+CFLAGS+="$(pkg-config --cflags gtk+-2.0) "
+CXXFLAGS+="$(pkg-config --cflags gtk+-2.0) "
 
 dnl assuming linux environment
 so_ext=".so.$ZEROPAD_RELEASE"
@@ -100,4 +107,5 @@
 
 echo "Configuration:"
 echo "  Debug build?	       $debug"
-echo "  x86-64 build?	       $cpu64"
\ No newline at end of file
+echo "  x86-64 build?	       $cpu64"
+echo "  Joystick build?	       $joystick"
