diff --git a/opengl/configure.ac b/opengl/configure.ac
index 83d2622..e4354c8 100644
--- a/opengl/configure.ac
+++ b/opengl/configure.ac
@@ -25,9 +25,9 @@ AC_SUBST(ZEROGS_AGE)
 AC_SUBST(ZEROGS_RELEASE)
 AC_SUBST(ZEROGS_SONAME)
 
-CFLAGS=" -I/opt/cg/include -L/opt/cg/lib -Wall -Wno-format -Wno-unused "
-CPPFLAGS=" -I/opt/cg/include -L/opt/cg/lib -Wall -Wno-format -Wno-unused "
-CXXFLAGS=" -I/opt/cg/include -L/opt/cg/lib -Wall -Wno-format -Wno-unused "
+CFLAGS=" -I/opt/nvidia-cg-toolkit/include -L/opt/nvidia-cg-toolkit/lib32 -Wall -Wno-format -Wno-unused "
+CPPFLAGS=" -I/opt/nvidia-cg-toolkit/include -L/opt/nvidia-cg-toolkit/lib32 -Wall -Wno-format -Wno-unused "
+CXXFLAGS=" -I/opt/nvidia-cg-toolkit/include -L/opt/nvidia-cg-toolkit/lib32 -Wall -Wno-format -Wno-unused "
 
 dnl Check for debug build
 AC_MSG_CHECKING(debug build)
@@ -36,14 +36,10 @@ AC_ARG_ENABLE(debug, AC_HELP_STRING([--enable-debug], [debug build]),
 if test "x$debug" == xyes
 then
  AC_DEFINE(_DEBUG,1,[_DEBUG])
- CFLAGS+="-g "
- CPPFLAGS+="-g "
- CXXFLAGS+="-g -fpermissive "
+ CXXFLAGS+="-fpermissive "
 else
  AC_DEFINE(NDEBUG,1,[NDEBUG])
- CFLAGS+="-O2 -msse2 "
- CPPFLAGS+="-O2 -msse2 "
- CXXFLAGS+="-O2 -msse2 -fno-regmove "
+ CXXFLAGS+="-fno-regmove "
 fi
 AC_MSG_RESULT($debug)
 
