diff -ur opengl.bak/configure.ac opengl/configure.ac
--- opengl.bak/configure.ac	2009-03-10 12:51:58.425023153 -0400
+++ opengl/configure.ac	2009-03-10 12:59:51.740273254 -0400
@@ -25,9 +25,9 @@
 AC_SUBST(ZEROGS_RELEASE)
 AC_SUBST(ZEROGS_SONAME)
 
-CFLAGS=" -I/opt/cg/include -L/opt/cg/lib -Wall -Wno-format -Wno-unused -g -pg "
-CPPFLAGS=" -I/opt/cg/include -L/opt/cg/lib -Wall -Wno-format -Wno-unused -g -pg "
-CXXFLAGS=" -I/opt/cg/include -L/opt/cg/lib -Wall -Wno-format -Wno-unused -g -pg "
+CFLAGS=" -I/opt/nvidia-cg-toolkit/include -L/opt/nvidia-cg-toolkit/lib -Wall -Wno-format -Wno-unused "
+CPPFLAGS=" -I/opt/nvidia-cg-toolkit/include -L/opt/nvidia-cg-toolkit/lib -Wall -Wno-format -Wno-unused "
+CXXFLAGS=" -I/opt/nvidia-cg-toolkit/include -L/opt/nvidia-cg-toolkit/lib -Wall -Wno-format -Wno-unused "
 
 dnl Check for debug build
 AC_MSG_CHECKING(debug build)
@@ -36,14 +36,9 @@
 if test "x$debug" == xyes
 then
  AC_DEFINE(_DEBUG,1,[_DEBUG])
- CFLAGS+="-g "
- CPPFLAGS+="-g "
- CXXFLAGS+="-g -fpermissive "
+ CXXFLAGS+=" -fpermissive "
 else
  AC_DEFINE(NDEBUG,1,[NDEBUG])
- CFLAGS+="-O0 -msse2"
- CPPFLAGS+="-O0 -msse2"
- CXXFLAGS+="-O0 -msse2"
 fi
 AC_MSG_RESULT($debug)
 
