--- a/configure.ac	2009-05-20 17:49:48.486570256 -0400
+++ b/configure.ac	2009-05-20 17:50:31.084353373 -0400
@@ -22,6 +22,17 @@
     [Define to be the subversion revision number])
 fi
 
+if test "x$GCC" = "xyes"; then
+  dnl get gcc version
+  AC_MSG_CHECKING([gcc version])
+  gccver=$($CC -dumpversion)
+  gccvermajor=$(echo $gccver | cut -d . -f1)
+  gccverminor=$(echo $gccver | cut -d . -f2)
+  gccverrevision=$(echo $gccver | cut -d . -f3)
+  gccvernum=$(expr $gccvermajor "*" 100 + $gccverminor "*" 10 + $gccverrevision)
+  AC_MSG_RESULT($gccver)
+fi
+
 AC_ARG_ENABLE(customcflags, AC_HELP_STRING([--enable-customcflags], [Use custom CFLAGS]),
 customcflags=$enableval,customcflags=no)
 
@@ -37,8 +48,11 @@
 
 WARNING_FLAGS="-Wall -Wno-format -Wno-unused-value"
 NORMAL_FLAGS=" -pipe -msse -O3 "
-# These optimizations seem to cause issues with GCC 4.3.3, so we'll turn them off.
-NORMAL_FLAGS+=" -fno-guess-branch-probability -fno-dse -fno-tree-dse "
+dnl These optimizations seem to cause issues with GCC 4.3.3, so we'll turn them off. 
+dnl I really need to run some tests and see if they are still needed.
+if test "$gccvernum" -ge "433"; then
+  NORMAL_FLAGS+=" -fno-guess-branch-probability -fno-dse -fno-tree-dse "
+fi
 
 DEBUG_FLAGS+=" -g -msse ${WARNING_FLAGS} "
 
