--- configure.ac.bak	2008-08-09 20:30:25.000000000 -0400
+++ configure.ac	2008-08-09 21:51:16.000000000 -0400
@@ -11,11 +11,6 @@
 dnl necessary for compiling assembly
 AM_PROG_AS
 
-CFLAGS=
-CPPFLAGS=
-CXXFLAGS=
-CCASFLAGS=
-
 dnl Check for debug build
 AC_MSG_CHECKING(debug build)
 AC_ARG_ENABLE(debug, AC_HELP_STRING([--enable-debug], [debug build]),
@@ -23,10 +18,7 @@
 if test "x$debug" == xyes
 then
  AC_DEFINE(_DEBUG,1,[_DEBUG])
- CFLAGS+=" -g "
- CPPFLAGS+=" -g "
- CXXFLAGS+=" -g "
- CCASFLAGS+=" -D_DEBUG -g "
+ CCASFLAGS+=" -D_DEBUG "
  MYOBJDIR="Debug"
 else
  AC_DEFINE(NDEBUG,1,[NDEBUG])
