--- Linux.old/GtkGui.c	2007-11-19 07:21:55.000000000 +0000
+++ Linux/GtkGui.c	2008-02-25 11:57:37.167773962 +0000
@@ -723,12 +723,9 @@
     strcpy(file, Config.PluginsDir); \
 	strcat(file, plugin); \
 	drv = SysLoadLibrary(file); \
-    getcwd(file, ARRAYSIZE(file)); /* store current dir */  \
-    chdir(Config.PluginsDir); /* change dirs so that plugins can find their config file*/  \
 	if (drv == NULL) return; \
 	conf = (src) SysLoadSym(drv, name); \
 	if (SysLibError() == NULL) conf(); \
-    chdir(file); /* change back*/       \
     SysCloseLibrary(drv);
 
 #define TestPlugin(src, confs, plugin, name) \
