--- a/configure.ac	2008-12-30 06:00:12.673899871 -0500
+++ b/configure.ac	2008-12-30 06:00:55.453903131 -0500
@@ -31,14 +31,11 @@
 if test "x$debug" == xyes
 then
  AC_DEFINE(_DEBUG,1,[_DEBUG])
- CFLAGS="-g "
- CPPFLAGS="-g "
- CXXFLAGS="-g "
 else
  AC_DEFINE(NDEBUG,1,[NDEBUG])
- CFLAGS="-O3 -fomit-frame-pointer "
- CPPFLAGS="-O3 -fomit-frame-pointer "
- CXXFLAGS="-O3 -fomit-frame-pointer "
+ CFLAGS=" -O3 -fomit-frame-pointer "
+ CPPFLAGS=" -O3 -fomit-frame-pointer "
+ CXXFLAGS=" -O3 -fomit-frame-pointer "
 fi
 AC_MSG_RESULT($debug)
 
