diff -ur a/src/Linux/Makefile b/src/Linux/Makefile
--- a/src/Linux/Makefile	2009-03-09 20:27:51.151324540 -0400
+++ b/src/Linux/Makefile	2009-03-09 20:28:19.446278009 -0400
@@ -1,10 +1,7 @@
-
-CC = gcc
-
 PLUGIN = libCDVDiso.so
 CFG = cfgCDVDiso
 MKISO = mkiso
-CFLAGS = -fPIC -Wall -g  -I.. -I. -D__LINUX__
+CFLAGS += -fPIC -Wall -g  -I.. -I. -D__LINUX__
 OBJS = ../CDVDisop.o Config.o Linux.o ../libiso.o
 CFGOBJS = conf.o interface.o support.o ${OBJS}
 LIBS = -lz -lbz2 -lstdc++
@@ -22,7 +19,6 @@
 cfg: ${CFGOBJS}
 	rm -f ${CFG}
 	${CC} ${CFLAGS} ${CFGOBJS} -o ${CFG} ${CFGLIBS}
-	strip ${CFG}
 
 clean: 
 	rm -f ${OBJS} ${DEPS} ${PLUGIN} ${CFGOBJS} ${CFG}
