--- Makefile.bak	2008-08-10 04:34:13.000000000 -0400
+++ Makefile	2008-08-10 04:36:36.000000000 -0400
@@ -21,8 +21,6 @@
 	../convert.h ../ini.h ../version.h
 
 
-CC = gcc
-
 GTKFLAGS = $(shell pkg-config --cflags gtk+-2.0)
 
 //GTKFLAGS += -DG_DISABLE_DEPRECATED \
@@ -43,21 +41,19 @@
 
 plugin: $(PLUGINOBJS) $(SHAREDOBJS)
 #	rm -f $(PLUGIN)
-	$(CC) -shared -Wl,-soname,$(PLUGIN) $(PLUGINFLAGS) $(PLUGINLIBS) \
+	$(CC) -shared -Wl,-soname,$(PLUGIN) $(PLUGINFLAGS) $(CFLAGS) $(LDFLAGS) $(PLUGINLIBS) \
 	$(PLUGINOBJS) $(SHAREDOBJS) -o $(PLUGIN)
-	strip --strip-unneeded --strip-debug $(PLUGIN)
 
 cfg: $(CFGOBJS) $(SHAREDOBJS)
 #	rm -f $(CFG)
-	$(CC) $(CFGFLAGS) $(GTKFLAGS) $(CFGLIBS) $(GTKLIBS) \
+	$(CC) $(CFGFLAGS) $(GTKFLAGS) $(CFLAGS) $(LDFLAGS) $(CFGLIBS) $(GTKLIBS) \
 	$(CFGOBJS) $(SHAREDOBJS) -o $(CFG)
-	strip $(CFG)
 
 $(PLUGINOBJS) $(SHAREDOBJS): %.o: %.c
-	$(CC) $(PLUGINFLAGS) -c $< -o $@
+	$(CC) $(PLUGINFLAGS) $(CFLAGS) -c $< -o $@
 
 $(CFGOBJS): %.o: %.c
-	$(CC) $(CFGFLAGS) $(GTKFLAGS) -c $< -o $@
+	$(CC) $(CFGFLAGS) $(GTKFLAGS) $(CFLAGS) -c $< -o $@
 
 .PHONY : clean allclean
 clean:
