diff -ur Linux.old/ConfigDlg.cpp Linux/ConfigDlg.cpp
--- Linux.old/ConfigDlg.cpp	2008-11-08 10:27:23.350010796 -0500
+++ Linux/ConfigDlg.cpp	2008-11-08 10:27:34.110048739 -0500
@@ -50,13 +50,10 @@
 	strcat(file, plugin); 
 
 	drv = SysLoadLibrary(file);
-	getcwd(file, ARRAYSIZE(file)); /* store current dir */  
-	chdir(Config.PluginsDir); /* change dirs so that plugins can find their config file*/  
 	if (drv == NULL) return;
 
 	conf = (void (*)()) SysLoadSym(drv, name);
 	if (SysLibError() == NULL) conf(); 
-	chdir(file); /* change back*/      
 	SysCloseLibrary(drv);
 }
 
