--- configure.ac	2008-11-23 16:09:56.291356952 -0500
+++ /root/configure.ac	2008-11-23 16:09:15.449544075 -0500
@@ -127,6 +127,13 @@
 AC_CHECK_HEADER([libintl.h], [AC_DEFINE(ENABLE_NLS,1,[__x86_64__])])
 fi
 
+AC_ARG_ENABLE(local-inis, AC_HELP_STRING([--enable-local-inis], [Enable local plugin inis]),
+localinis=$enableval, local-inis=yes)
+if test "x$localinis" == xyes
+then
+AC_DEFINE(LOCAL_PLUGIN_INIS)
+fi
+
 AC_OUTPUT([
  Makefile
  DebugTools/Makefile
@@ -149,4 +156,5 @@
 echo "  Dev build?	           $devbuild"
 echo "  Force sse3?	           $sse3"
 echo "  Recompilers enabled?   $recbuild"
-echo "  nls support?	       $nls"
\ No newline at end of file
+echo "  nls support?	       $nls"
+echo "  local plugin inis?     $localinis"
