--- configure.ac.bak	2008-11-23 16:00:10.858823992 -0500
+++ configure.ac	2008-11-23 16:00:06.378852799 -0500
@@ -22,14 +22,9 @@
     [Define to be the subversion revision number])
 fi
 
-CFLAGS=
-CPPFLAGS=
-CXXFLAGS=
-CCASFLAGS=
-
 WARNING_FLAGS="-Wall -Wno-format -Wno-unused-value"
 NORMAL_FLAGS=" -pipe -O3 -fomit-frame-pointer ${WARNING_FLAGS}"
-DEBUG_FLAGS=" -g -O0 ${WARNING_FLAGS}"
+DEBUG_FLAGS=" -O0 ${WARNING_FLAGS}"
 
 dnl Check for debug build
 AC_MSG_CHECKING(debug build)
