Index: GSnull/configure.ac
===================================================================
--- GSnull/configure.ac	(revision 1736)
+++ GSnull/configure.ac	(working copy)
@@ -30,15 +30,15 @@
 if test "x$debug" == xyes
 then
  AC_DEFINE(PCSX2_DEBUG,1,[PCSX2_DEBUG])
- CFLAGS+="-g -fPIC -Wall  -Wno-unused-value -m32 "
- CPPFLAGS+="-g -fPIC -Wall  -Wno-unused-value -m32 "
- CXXFLAGS+="-g -fPIC -Wall  -Wno-unused-value -m32 "
+ CFLAGS+="-fPIC -Wall  -Wno-unused-value -m32 "
+ CPPFLAGS+="-fPIC -Wall  -Wno-unused-value -m32 "
+ CXXFLAGS+="-fPIC -Wall  -Wno-unused-value -m32 "
  CCASFLAGS+=" -m32 "
 else
  AC_DEFINE(NDEBUG,1,[NDEBUG])
- CFLAGS+="-O3 -fomit-frame-pointer -msse -msse2 -fPIC -Wall  -Wno-unused-value -m32 "
- CPPFLAGS+="-O3 -fomit-frame-pointer -msse -msse2 -fPIC -Wall  -Wno-unused-value -m32 "
- CXXFLAGS+="-O3 -fomit-frame-pointer -msse -msse2 -fPIC -Wall  -Wno-unused-value -m32 "
+ CFLAGS+="-msse2 -fPIC -Wall  -Wno-unused-value -m32 "
+ CPPFLAGS+="-msse2 -fPIC -Wall  -Wno-unused-value -m32 "
+ CXXFLAGS+="-msse2 -fPIC -Wall  -Wno-unused-value -m32 "
  CCASFLAGS+=" -m32 "
 fi
 AM_CONDITIONAL(DEBUGBUILD, test x$debug = xyes)
@@ -80,4 +80,4 @@
 
 echo "Configuration:"
 echo "  Debug build?	       $debug"
-echo "  Dev build?	       $devbuild"
\ No newline at end of file
+echo "  Dev build?	       $devbuild"
